CMakeLists.txt

```diff
@@ -1113,6 +1113,12 @@ add_subdirectory(utils)
 
 add_subdirectory(userdocs)
 
+file(GLOB LOCALIZATION_FILES
+  "include/swift/AST/diagnostics/*.yaml"
+)
+
+file(COPY ${LOCALIZATION_FILES} DESTINATION "share/diagnostics")
```

Comment:
I think we might want to do something like:

```
file(GLOB LOCALIZATION_FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/include/swift/AST/diagnostics/*.yaml)

if (SWIFT_BUILT_STANDALONE)
  file(COPY ${LOCALIZATION_FILES} DESTINATION "share/swift/diagnostics")
else()
  swift_install_in_component(FILES ${LOCALIZATION_FILES}
                             DESTINATION "share/swift/diagnostics"
                             COMPONENT compiler)
endif()
```

But I'm not a CMake expert so any input is welcome. /cc @shahmishal

Reply:
Globs are generally to be avoided in CMake.  The glob executes once and changes will _not_ be detected.  It really is better to explicitly enumerate the files.

Reply:
What is the recommended way to copy files then?